### PR TITLE
support node4

### LINKED ISSUE #13: Various Node Versions Support
This is probably in the early stages, so I apologize if this issue is premature. From the documentation, it looks like the intent is to support node 4 <=, but currently isn't transpiled down to support Node 4.

##### /bin/babel-upgrade
```
const { isAcceptedNodeVersion, writePackageJSON, writeBabelRC } = require("../src");
      ^
SyntaxError: Unexpected token {
```

##### /src/index.js:59
```
async function writePackageJSON() {
      ^^^^^^^^
SyntaxError: Unexpected token function
```

##### /src/packageData.js:243
```
);
^
SyntaxError: Unexpected token )
```

Otherwise, everything has worked great in node 8 & 9 .

### PR DESCRIPTION
**Pull request by abouthiroppy**
Fixed #13.

Make use of `@babel/preset-env` and specify Node4.

I do not think the name of `src/bin.js` is very good.
So I want to change the name of `src/index.js` and rename `src/bin.js` to `src/index.js`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +30,10 @@
     "write-pkg": "^3.1.0"
   },
   "devDependencies": {
+    "@babel/cli": "^7.0.0-beta.40",
+    "@babel/core": "^7.0.0-beta.40",
+    "@babel/polyfill": "^7.0.0-beta.40",
```
Comment (hzoo) [inline package.json]: this will need to be a dep

**Comment by abouthiroppy** [comment]
@hzoo Updated:)

**Comment by hzoo** [comment]
Haha we need babel-core too right..

**Comment by abouthiroppy** [comment]
@hzoo Is this okay? https://github.com/babel/babel-upgrade/pull/16/commits/b1dd1ec91dc15038f040022be3e376fffeb86383

**Comment by hzoo** [comment]
Yeah in order to know this actually works we need to add travis, so I did that ^, lets see

**Comment by abouthiroppy** [comment]
Thanks for adding Travis;)

**Comment by hzoo** [comment]
finally.. 😄

**Pull request merged**